pkg/operators/ebpf/extrainfo.go

```diff
@@ -108,8 +118,22 @@ func (i *ebpfInstance) addExtraInfo(gadgetCtx operators.GadgetContext) error {
 		ContentType: "application/json",
 		Content:     []byte(variablesJson),
 	}
+	ebpfInfo.Data["ebpf.relocations"] = &api.GadgetInspectAddendum{
+		ContentType: "application/json",
+		Content:     []byte("[]"),
+	}
 
 	gadgetCtx.SetVar("extraInfo.ebpf", ebpfInfo)
 
 	return nil
 }
+
+func GetRelocations(insns asm.Instructions) {
+	iter := insns.Iterate()
+	for iter.Next() {
+		relo := btf.CORERelocationMetadata(iter.Ins)
+		if relo != nil {
+			fmt.Println("Relocation:", relo, "; Line: ", iter.Ins.Source())
+		}
```

Comment:
We should print lines without relocation too, for context. Something like:

```suggestion
		source := iter.Ins.Source()
		if source != "" {
			fmt.Printf("; %s\n", source)
		}
		relo := btf.CORERelocationMetadata(iter.Ins)
		if relo != nil {
			fmt.Println("; %v\n", relo)
		}
		fmt.Printf("%v\n", iter.Ins)
```